doc/source/ray-overview/installation.rst

```diff
@@ -52,11 +52,12 @@ You can install the nightly Ray wheels via the following links. These daily rele
 ====================  ====================  =======================
        Linux                 MacOS          Windows (beta)
 ====================  ====================  =======================
-`Linux Python 3.10`_  `MacOS Python 3.10`_  `Windows Python 3.10`_
-`Linux Python 3.9`_   `MacOS Python 3.9`_   `Windows Python 3.9`_
-`Linux Python 3.8`_   `MacOS Python 3.8`_   `Windows Python 3.8`_
-`Linux Python 3.7`_   `MacOS Python 3.7`_   `Windows Python 3.7`_
-`Linux Python 3.6`_   `MacOS Python 3.6`_
+`Linux Python 3.11`_  `MacOS Python 3.10`_  `Windows Python 3.10`_
```

Comment:
can we avoid this "off by one" format? maybe just put 3.11 at the bottom (and label as experimental if necessary?)